src/app/components/room/room.service.js

```diff
@@ -101,6 +101,9 @@
     function doEnter(username) {
       var $$rootScope = $rootScope;
       var connection = null;
+      
+      //max number of participants who will join session unmuted
+      var joinUnmutedLimit = 3; 
```

Comment:
IMO, this should be a configuration option in `config.json`. If set to null (or undef), it would mean "never start muted".